src/CMakeLists.txt

```diff
@@ -72,9 +72,11 @@ if ("${FIRESTARTER_BUILD_TYPE}" STREQUAL "FIRESTARTER_CUDA")
 		Nitro::log
 		nlohmann_json::nlohmann_json
 		Threads::Threads
-		${CUDA_LIBRARIES}
-		${CUDA_CUBLAS_LIBRARIES}
-		${CUDA_curand_LIBRARY}
+                CUDA::cuda_driver
+                CUDA::cudart
+                CUDA::cufft
+                CUDA::curand
+                CUDA::cublas
```

Comment:
It seems like you are using spaces, where everything else uses tabs, could you change that?